plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/delete/DeleteFile.java

```diff
@@ -103,11 +121,25 @@ public List<Integer> equalityFieldIds()
         return equalityFieldIds;
     }
 
+    @JsonProperty
+    public Map<Integer, byte[]> getLowerBounds()
+    {
+        return lowerBounds;
+    }
+
+    @JsonProperty
+    public Map<Integer, byte[]> getUpperBounds()
+    {
+        return upperBounds;
+    }
+
     public long getRetainedSizeInBytes()
     {
         return INSTANCE_SIZE
                 + estimatedSizeOf(path)
-                + estimatedSizeOf(equalityFieldIds, ignored -> SIZE_OF_INT);
+                + estimatedSizeOf(equalityFieldIds, ignored -> SIZE_OF_INT)
+                + estimatedSizeOf(lowerBounds, entry -> SIZE_OF_INT, buffer -> buffer.length)
+                + estimatedSizeOf(upperBounds, entry -> SIZE_OF_INT, buffer -> buffer.length);
```

Comment:
> `buffer -> buffer.length`

this isn't perfect. empty byte[] still uses memory
and single-element byte[] uses more than 1 byte

use `io.airlift.slice.SizeOf#sizeOf(byte[])`